utils/jsonConfigWriter.py

```diff
@@ -0,0 +1,66 @@
+import json
```

Comment:
Just to the filename: the lowercase start of the file looks strange. Not pythonic. Can you take a look in the standard python distribution how they names files like this.

Reply:
changed in latest commit